model/bin/ww3_from_ftp.sh

```diff
@@ -53,7 +53,7 @@ else
   echo -e ' Not deleting tar file.' 
 fi
 
-echo -e "\n\n Files were copied from the data_regtests to the regtets folder."
+echo -e "\n\n Files were copied from the data_regtests to the regstets folder."
```

Comment:
regtests

Reply:
@JessicaMeixner-NOAA at least I counted the number of letters :smile: